src/main/java/org/wildfly/security/http/HttpServerRequest.java

```diff
@@ -63,6 +64,13 @@
     SSLSession getSSLSession();
 
     /**
+     * Get the peer certificates established on the connection.
+     *
+     * @return the peer certificates established on the connection or {@code null} if none available.
+     */
+    Certificate[] getPeerCertificates();
```

Comment:
I really think we should be using ```X509Certificate[]``` everywhere.  ```org.wildfly.security.x500.X500#asX509CertificateArray()``` can be used to safely narrow the array.